var/spack/repos/builtin/packages/git/package.py

```diff
@@ -69,6 +76,7 @@ class Git(Package):
 
     def install(self, spec, prefix):
         env['LDFLAGS'] = "-L%s" % spec['gettext'].prefix.lib + " -lintl"
+        env['NO_GETTEXT'] = "YesPlease"
```

Comment:
in other words, you don't need this change, right? Then please remove it :smile: